community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/checkpoint/Rush.java

```diff
@@ -17,22 +17,17 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.io.pagecache.impl.muninn;
+package org.neo4j.kernel.impl.transaction.log.checkpoint;
 
 /**
- * This runnable continuously flushes dirty pages in the background, such that future calls to
- * {@link MuninnPageCache#flushAndForce()} have less work to do.
+ * An object that represents a temporary period of rushing for the check pointer.
+ * @see CheckPointFlushControl
  */
-final class FlushTask extends BackgroundTask
+public interface Rush extends AutoCloseable
```

Comment:
Even while using Rush concept we able to `rush` i believe it's just bring additional complexity since same can be achieved just by resetting current limit using checkpointFlushControl